example/requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 html5lib==0.9999999
-django==1.10
+django==1.9
```

Comment:
proč z 1.10 na 1.9?

Reply:
Protože jsem to testoval pro 1.9 a nechal jsem to tam, má tam být 1.11, jelikož 2 ještě nepoužíváme díky